package.json

```diff
@@ -26,6 +26,7 @@
     "node": "18.15.0"
   },
   "dependencies": {
+    "@chakra-ui/icons": "^2.0.19",
```

Comment:
Ser ikke ut som at denne blir brukt noe sted, men ser uansett ut i andre pull requests her som at en annen løsning for ikoner skal brukes

```suggestion
```